docs/en/observability/cloud-monitoring/aws/monitor-aws-waf-firehose.asciidoc

```diff
@@ -0,0 +1,141 @@
+[[monitor-aws-waf-firehose]]
+= Monitor Web Application Firewall (WAF) logs
+
+++++
+<titleabbrev>Monitor WAF logs</titleabbrev>
+++++
+
+In this section, you'll learn how to send AWS WAF events from AWS to your Elastic stack using Amazon Data Firehose.
+
+You will go through the following steps:
+
+- Select a WAF-compatible resource (CloudFront distribution)
+- Create a delivery stream in Amazon Data Firehose
+- Create a web Access Control List (ACL) to generate WAF logs
+- Set up logging to forward the logs to the Elastic stack using a Firehose stream
+- Visualize your WAF logs in {kib}
+
+[discrete]
+[[firehose-cloudtrail-prerequisites]]
+== Before you begin
+
+We assume that you already have:
+
+- An AWS account with permissions to pull the necessary data from AWS.
+- A deployment using our hosted {ess} on {ess-trial}[{ecloud}]. The deployment includes an {es} cluster for storing and searching your data, and {kib} for visualizing and managing your data. AWS Data Firehose works with Elastic Stack version 7.17 or greater, running on Elastic Cloud only.
+
+IMPORTANT: Make sure the deployment is on AWS, because the Amazon Data Firehose delivery stream connects specifically to an endpoint that needs to be on AWS.
+
+[discrete]
+[[firehose-waf-step-one]]
+== Step 1: Install AWS integration in {kib}
+
+. In {kib}, navigate to *Management* > *Integrations* and browse the catalog to find the Amazon Data Firehose integration.
+
+. Navigate to the *Settings* tab and click *Install Amazon Data Firehose assets*.
+
+[discrete]
+[[firehose-waf-resource-step-two]]
+== Step 2: Select a WAF-compatible resource
+
+You can create and attach a web ACL to several AWS resources:
+
+- CloudFront distribution
+- Application Load Balancers
+- Amazon API Gateway REST APIs
+- Amazon App Runner services
+- AWS AppSync GraphQL APIs
+- Amazon Cognito user pools
+- AWS Verified Access Instances
+
+If you don't have an existing resource for testing, you can create a simple CloudFront distribution.
+
+[discrete]
+[[firehose-waf-step-three]]
+== Step 3: Create a delivery stream in Amazon Data Firehose
+
+. Go to the https://console.aws.amazon.com/[AWS console] and navigate to Amazon Data Firehose.  
+
+. Click *Create Firehose stream* and choose the source and destination of your Firehose stream. Unless you are streaming data from Kinesis Data Streams, set source to `Direct PUT` and destination to `Elastic`. 
+
+. Provide a meaningful *Firehose stream name* that will allow you to identify this delivery stream later.
```

Comment:
WAF restricts Firehose stream names. The warning "Your Firehose name must start with the prefix `aws-waf-logs—`, or it will not show up later." should go here.